docs/appendices/release-notes/unreleased.rst

```diff
@@ -116,5 +116,10 @@ Fixes
   creating a subscription with bad credentials and ``pg_tunnel`` followed by
   re-creating it second time with the same name and valid credentials.
 
+- Fixed an issue with ``VARCHAR`` and ``BIT`` columns with specified length
+  used in primary key, generated or default column expression which led to lose
+  of the length property after triggering ``ALTER TABLE``.
```

Comment:
```suggestion
- Fixed an issue with ``VARCHAR`` and ``BIT`` columns with a length
  limited used in primary key, generated or default column
  expression. A ``ALTER TABLE`` statement  removed the length limit
  from such columns.
```

not sure about this - but the "which led to lose of the length property" sounds somehow wrong